src/shared/utils/index.ts

```diff
@@ -8,6 +8,7 @@ import logger from './logger'
 
 const log = logger({ name: 'utils' })
 const HTTPS_PORT = 51234
+const RIPPLE_EPOCH_DIFF = 0x386d4380
```

Comment:
This should be in decimal so it's easier to read (and matches the xrpl.js/Explorer code).